src/main/java/com/example/bookstoreproject/api/book/UserController.java

```diff
@@ -0,0 +1,49 @@
+package com.example.bookstoreproject.api.book;
+
+import com.example.bookstoreproject.domain.user.UserService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import static com.example.bookstoreproject.api.book.UserDTOMapper.toUserDTO;
+import static com.example.bookstoreproject.api.book.UserDTOMapper.toUserDTOs;
+import static com.example.bookstoreproject.domain.user.UserDomainMapper.toUser;
+
+@RestController
+@RequestMapping("/api/v1/users")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+
+    @GetMapping
+    public List<UserDTO> findAll() {
+        return toUserDTOs(userService.findAll());
+    }
+
+    @GetMapping("/{id}")
+    public UserDTO findUserById(@PathVariable UUID id) {
+        return toUserDTO(userService.findUserById(id));
+    }
+
+    @PostMapping("/create")
+    public UserDTO createUser(final @RequestBody UserDTO userDTO) {
+        return toUserDTO(userService.createUser(toUser(userDTO)));
+    }
+
+    @PatchMapping("/update/{id}")
```

Comment:
Check có đúng là PATCH không nha. Và mình dùng HTTP Method để phân biệt các phương thức nên không cần thêm các tiền tố như create, delete, update ... 

Mình chỉ thêm trong các trường hợp đặc biệt. Common thế này không cần

Và cũng không cần thêm `/` ở đâu, mặc định có sẵn rồi.

```suggestion
    @PatchMapping("{id}")
```

Reply:
Dạ. DONE anh